plugin/latexlivepreview.vim

```diff
@@ -184,14 +184,23 @@ EEOOFF
                         \ l:tmp_root_dir . '/' . bib_fn)                                " TODO: may fail if same bibfile names in different dirs
         endfor
 
+        if exists('g:livepreview_biber') && g:livepreview_biber > 0
+            let s:bibexec = 'biber *.bcf'
+        else
+            let s:bibexec = 'bibtex *.aux'
+        endif
```

Comment:
Other configuration flags ( `engine`, `previewer` ) can set the binary directly, we could try to do something similar for bibliography 

```suggestion
        if (s:bibengine == 'biber')
            let s:bibexec = 'biber ' . l:tmp_root_dir . '*.bcf'
        else
            let s:bibexec = s:bibengine . ' ' . l:tmp_root_dir . '*.aux'
        endif
```

---

We should also set this in the `Initialize` function ( found this in https://stackoverflow.com/a/33775128, to be tested though )

```vim
    " Get the bib engine
    let s:bibengine = get(g:, 'livepreview_bibengine', 'bibtex')
```

https://github.com/xuhdev/vim-latex-live-preview/blob/855c30915547c7e9c19b7e6efb528730c071d2e8/plugin/latexlivepreview.vim#L230-L252

Reply:
I went back and forth on this, but I finally settled on disagreeing with this suggested change for the following reasons:
1. `pdflatex` and `xelatex` are drop in replacements of each other, as are `evince` and `okular`, however `bibtex` and `biber` have usages divergent enough to break things. They will be a binary choice for the foreseeable future.
2. I don't think we should ask the user to spell something when they can simply select instead.

Reply:
I did however use `get()` with some added validation for both `g:livepreview_engine` and `g:livepreview_previewer` which now gives a warning if no valid executable is available for either of those roles.